### PR TITLE
[BEAM-6718] Fix BigQuery SQL postcommit

### PR DESCRIPTION
**Pull request by reuvenlax**

### CODE REVIEW

**Comment by reuvenlax** [comment]
Run SQL PostCommit

**Comment by reuvenlax** [comment]
Run SQL PostCommit

**Comment by reuvenlax** [comment]
Run SQL PostCommit

**Comment by reuvenlax** [comment]
Run Java PreCommit

**Comment by reuvenlax** [comment]
Run JavaPortabilityApi PreCommit

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/AvroUtils.java [hunk 1]
```diff
@@ -17,14 +17,22 @@
...
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Set;
 import org.apache.beam.sdk.schemas.Schema.Field;
 import org.apache.beam.sdk.schemas.Schema.TypeName;
 import org.joda.time.Instant;
 
 /** Utils to help convert Apache Avro types to Beam types. */
 public class AvroUtils {
+  // TODO: BigQuery shouldn't know about SQL internal logical types.
```
Comment (kennknowles) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/AvroUtils.java]: Can you file a Jira about this. It seems like a pretty big deal.

**Comment by kennknowles** [comment]
run java precommit

**Comment by kennknowles** [comment]
```
[ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/beam_PreCommit_Java_Phrase/src/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/AvroUtils.java:20: You are using raw guava, please use vendored guava classes. [ForbidNonVendoredGuava]
[ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/beam_PreCommit_Java_Phrase/src/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/AvroUtils.java:36:3: Missing a Javadoc comment. [JavadocMethod]

```

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/AvroUtils.java [hunk 2]
```diff
@@ -17,14 +17,22 @@
  */
 package org.apache.beam.sdk.io.gcp.bigquery;
 
+import com.google.common.collect.ImmutableSet;
```
Comment (kennknowles) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/AvroUtils.java]: vendored!

Comment (kennknowles) [reply sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/AvroUtils.java]: @iemejia nice catch :-)

**Comment by kennknowles** [comment]
run java precommit

**Comment by kennknowles** [comment]
Failure in Cassandra test. Since this restores postcommit signal, I am going to merge w/ squash.

**Pull request merged**